stacks/api/bucket/bucket.controller.spec.ts

```diff
@@ -143,35 +144,9 @@ describe("Bucket acceptance", () => {
 
     it("should get all buckets", async () => {
       //add buckets
-      const firstBucket = {
-        _id: new ObjectId(),
-        title: "New Bucket",
-        description: "Describe your new bucket",
-        icon: "view_stream",
-        primary: "title",
-        readOnly: false,
-        properties: {
-          title: {
-            type: "string",
-            title: "title",
-            description: "Title of the row",
-            options: {position: "left", visible: true}
-          },
-          description: {
-            type: "string",
-            title: "description",
-            description: "Description of the row",
-            options: {position: "right", visible: true}
-          }
-        }
-      };
-      const secondBucket = {
-        _id: new ObjectId(),
-        title: "New Bucket",
-        description: "Describe your new bucket",
-        icon: "view_stream",
-        primary: "title",
-        readOnly: false,
+      const firstBucket = (await req.post("/bucket", bucket)).body;
```

Comment:
```suggestion
      const {body: firstBucket} = await req.post("/bucket", bucket);
```

Reply:
IMHO, the suggested change would make it much more readable. What do you think ?

Reply:
There are too many lines which are like this. We should change them as well if you agree with me too.

Reply:
If you do not know the **destructuring assignment** ( I didn't know it before JavaScript ), my style is more readable but it won't take too much time, I can make this change suggestions of course. :)